### DISCUSSION TITLE
Improvements to the "Getting Started" guide

### DISCUSSION THREAD

**Discussion by dventimihasura** [Netflix/dgs-framework#1830] [General]
Hey folks.  Can I suggest some minor improvements to the [Getting Started guide](https://netflix.github.io/dgs/getting-started/)?  I would make a PR but I haven't yet fond the repo where that content is stored. 

1. In [Implement a Data Fetcher](https://netflix.github.io/dgs/getting-started/#implement-a-data-fetcher) it says to "create two new classes `example.ShowsDatafetcher` and `Show`, but if the user created the demo app with the Spring Initialzr with the defaults and according to the image, the package will be `com.example.demo` not `example`.
2. In that same place, there's a typo in `ShowsDatafetcher` with a lower-case "f".  Later in the code-block the class is `ShowsDataFetcher`.
3. The code block omits the import statements.  Granted, IDEs typically will help the user import those, but even then there can be some ambiguity.  It might be worthwhile to spell that out explicitly.
4. In [Test the app with GraphiQL](https://netflix.github.io/dgs/getting-started/#test-the-app-with-graphiql) it says, "Start the application and open a browser to http://localhost:8080/graphiql."  Given that the guide urged readers to use the Spring Initialzr, it's fair to say some or many of those readers will be new to Spring and so won't know how to start the application.  Again, IDEs may help with this, but it might be worthwhile to spell it out explicitly.
5. If there are generated JavaDoc pages for the framework, it might be worth linking them in somewhere.  That's always appreciated.

Again, I'll poke around and see if I can find the repo for these docs and will try to make a PR.  Until then, it's just food for thought.  Thanks so much!  

Cheers,
David

**Comment by congotej** [comment]
Hi @dventimihasura! Thanks so much for bringing this up! Here's the repo for the docs: https://github.com/Netflix/dgs. A PR would be the quickest way to get these changes in, otherwise I'll go ahead and prioritize it on our end so that we can get this fixed sooner rather than later.

**Reply by dventimihasura** [reply]
You bet.  Will do.  Thanks!

**Reply by dventimihasura** [reply]
hey @congotej I went ahead and made that PR.
